python/paddle/trainer_config_helpers/layers.py

```diff
@@ -4230,8 +4251,9 @@ def rnn_step(input):
     - machine translation : demo/seqToseq/translation/gen.conf \
                             demo/seqToseq/seqToseq_net.py
 
-    :param name: Name of the recurrent unit that generates sequences.
-    :type name: base string
+    :param name: The name of the recurrent unit that generates sequences.
```

Comment:
that generates sequences. --> that is responsible for generating sequences.

Reply:
Done